ci/infra/azure/terraform.tfvars.example

```diff
@@ -0,0 +1,103 @@
+# AZURE location
+# A list of location names can be found here:
```

Comment:
To keep the tfvars clean, could we put this block in Doc instead here?
```
# A list of location names can be found here:
# https://azure.microsoft.com/en-in/global-infrastructure/locations/
# https://azure.microsoft.com/en-us/global-infrastructure/regions/
# https://docs.microsoft.com/en-us/azure/availability-zones/az-region
```

Reply:
removed